app/src/interfaces/file/index.ts

```diff
@@ -16,14 +16,44 @@ export default defineInterface({
 			name: '$t:interfaces.system-folder.folder',
 			type: 'uuid',
 			meta: {
-				width: 'full',
+				width: 'half',
 				interface: 'system-folder',
 				note: '$t:interfaces.system-folder.field_hint',
 			},
 			schema: {
 				default_value: undefined,
 			},
 		},
+		{
+			field: 'type',
+			name: '$t:interfaces.system-folder.type',
+			type: 'string',
+			meta: {
+				width: 'half',
+				interface: 'select-dropdown',
+				note: '$t:interfaces.system-folder.type_hint',
+				options: {
+					allowNone: true,
+					choices: [
+						{
+							text: '$t:interfaces.system-folder.type_image',
+							value: 'image',
+						},
+						{
+							text: '$t:interfaces.system-folder.type_video',
```

Comment:
We should add the option for "Allow other", so you can specify specific mime types, instead of just "all images"